packages/vscode-extension/src/webview/components/SettingsDropdown.tsx

```diff
@@ -58,7 +60,22 @@ function SettingsDropdown({ project, isDeviceRunning, children, disabled }: Sett
               <KeybindingInfo commandName="RNIDE.openDevMenu" />
             </div>
           </DropdownMenu.Item>
-
+          <div className="dropdown-menu-item">
+            <span className="codicon codicon-device-mobile" />
+            <label className="switch-group-center" htmlFor="device-frame" style={{ width: "100%" }}>
+              <span>Show Device Frame</span>
+              <Switch.Root
+                style={{ transform: "scale(0.75)", marginLeft: "auto" }}
```

Comment:
@jakub-gonet The default size is too big for settings dropdown, and I added inline transform to reduce its size. Maybe I should create `switch-root--small` or just keep it?